### PR TITLE
VC-2957 wp codestyle

### PR DESCRIPTION
**Pull request by pavelthq**

### CODE REVIEW

File: visualcomposer/Framework/helpers.php [hunk 1]
```diff
@@ -335,6 +351,7 @@ function _vcCheckIsResponseBad($response)
             $responseCode = wp_remote_retrieve_response_code($response);
             $isRequestError = $responseCode !== 200;
             if ($isRequestError) {
+                // translators: %s: error message
```
Comment (OlegApanovich) [inline visualcomposer/Framework/helpers.php:354]: I can see %s in sprint format argument  // translators: %d: error response code  Looks more reasonable

Comment (pavelthq) [reply visualcomposer/Framework/helpers.php]: fixed

**Review by OlegApanovich — CHANGES_REQUESTED** [review]

File: visualcomposer/Helpers/AddonViews.php [hunk 2]
```diff
@@ -52,7 +53,6 @@ public function render($_path, $_args = [])
                 'args' => $_args,
             ]
         );
-        /** @noinspection PhpIncludeInspection */
         include($_path);
         $content = ob_get_clean();
```
[the remark this answers is not in the document]
Comment (pavelthq) [reply visualcomposer/Helpers/AddonViews.php:57]: these local variables are not in codestyle. and it is useful when debugging. just remove it from your inspection profile.

File: visualcomposer/Modules/Vendors/WpmlController.php [hunk 3]
```diff
@@ -250,22 +255,26 @@ protected function completeTranslationJobSaving($fields, $job)
...
             $class = 'notice notice-info';
             printf(
                 '<div class="%1$s"><p>%2$s</p></div>',
                 esc_attr($class),
                 sprintf(
-                    __(
-                        '<b>Visual Composer:</b> To complete WPML Translation Manager process for the Visual Composer supported pages you will need to run automatic posts update. <a href="%s">Update Posts</a>',
+                // translators: %1$s: <strong>, %2$s: </strong>, %3$s: url to automatic post updates page
+                    esc_html__(
+                        '%1$sVisual Composer:%2$s To complete WPML Translation Manager process for the Visual Composer supported pages you will need to run automatic posts update. <a href="%3$s">Update Posts</a>',
```
[the remark this answers is not in the document]
Comment (pavelthq) [reply visualcomposer/Modules/Vendors/WpmlController.php]: fixed

File: visualcomposer/Modules/Vendors/WpmlController.php [hunk 4]
```diff
@@ -125,6 +125,8 @@ protected function prepareTranslationJobData($package)
                 continue;
             }
             // Make the magic
+            // Content Encoded in Base64 as it is safe to use in URL and JSON
```
[the remark this answers is not in the document]
Comment (pavelthq) [reply visualcomposer/Modules/Vendors/WpmlController.php:128]: we can do that later, like move to helper. but not yet, as it is out of scope. this comment is just informative for wp com codestyle.

File: visualcomposer/resources/views/vendors/dashboard.php [hunk 5]
```diff
@@ -96,13 +96,13 @@ class="button button-primary">
                 $categories = implode(', ', $categoryArray);
                 ?>
                 <li>
-                    <a href="<?php echo $permalink . $utmHelper->get('wpdashboard-news-blog-post'); ?>" class="rsswidget" rel="noopener noreferrer" title=" <?php echo $title; ?>"
+                    <a href="<?php echo esc_url($permalink . $utmHelper->get('wpdashboard-news-blog-post')); ?>" class="rsswidget" rel="noopener noreferrer" title=" <?php echo esc_attr($itemTitle); ?>"
                             target="_blank">
-                        <?php echo $title; ?>
+                        <?php echo esc_html($title); ?>
```
[the remark this answers is not in the document]
Comment (pavelthq) [reply visualcomposer/resources/views/vendors/dashboard.php]: fixed

File: visualcomposer/Helpers/Localizations.php [hunk 6]
```diff
@@ -338,16 +338,18 @@ public function getLocalizations()
                 'visualcomposer'
             ),
             'newPluginVersionIsAvailable' => sprintf(
+                // translators: %s: Plugin url, %s: Plugin version
```
[the remark this answers is not in the document]
Comment (pavelthq) [reply visualcomposer/Helpers/Localizations.php]: fixed

File: visualcomposer/Helpers/AssetsShared.php [hunk 7]
```diff
@@ -11,15 +11,32 @@
 use VisualComposer\Framework\Container;
 use VisualComposer\Framework\Illuminate\Support\Helper;
 
+/**
+ *
+ */
 class AssetsShared extends Container implements Helper
 {
+    /**
+     * @var null
```
[the remark this answers is not in the document]
Comment (pavelthq) [reply visualcomposer/Helpers/AssetsShared.php]: fixed

File: visualcomposer/Helpers/Assets.php [hunk 8]
```diff
@@ -107,6 +117,11 @@ public function getFilePath($filename = '')
         return $path;
     }
 
+    /**
+     * @param $filePath
+     *
+     * @return mixed|string
```
[the remark this answers is not in the document]
Comment (pavelthq) [reply visualcomposer/Helpers/Assets.php]: always string will be correct

File: visualcomposer/Helpers/Assets.php [hunk 9]
```diff
@@ -13,6 +13,9 @@
 use VisualComposer\Framework\Container;
 use WP_Query;
 
+/**
+ *
```
[the remark this answers is not in the document]
Comment (pavelthq) [reply visualcomposer/Helpers/Assets.php]: fixed

**Review by veidenbaums — APPROVED** [review]

**Review by OlegApanovich — APPROVED** [review]

**Pull request merged**